### DISCUSSION TITLE
NATS Dashboard showing that more than 100% of cpu is used, although that's not the case

### DISCUSSION THREAD

**Discussion by albionb96** [nats-io/nats-server#5053] [Q&A] [answered]
I deployed a nats cluster and started some loadtests on it, I saw on the nats grafana dashboard that the cpu load is far beyond 100%,
![image](https://github.com/nats-io/nats-server/assets/49485798/099ecc9b-e92b-4019-9a4e-cf07b14e4186)

 but I don't know how this metric works, because the kuberntes nodes where the nats replicas run, have still very much free cpu. only 1.4 cores out of 6 are being used.

**Comment by Jarema** [comment, marked as answer]
Not 100% sure, but isn't this Grafana metric reporting cumulative CPU utilisation per core? That would mean that 140% is 1.4 Cores, which aligns with what you see.

**Reply by albionb96** [reply]
If that's true than that would make sense. But somebody needs to confirm this.

**Reply by ripienaar** [reply]
`nats server info` will show cores and usage, if you show that you can confirm.

**Reply by albionb96** [reply]
nast server info shows this for one of the nodes: 
![image](https://github.com/nats-io/nats-server/assets/49485798/26bd6c8d-c423-4968-8180-077a10a1683c)
I forgot to mention that actually the kuberntes nodes where the nats pods live have 12 cores (@2Ghz), and not 6, that was my mistake.

I do not believe that 64% of the cpu is being used, as I said the overall kuberntes node cpu load (including other pods also other than nats replica) is at around 1.4 cores out of 12.

the other two nats server replicas show lower cpu load 16% and 6 %.
